docs/install/pxe-boot-install.md

```diff
@@ -108,11 +108,13 @@ boot
 This assumes the iPXE script is stored in `/usr/share/nginx/html/harvester/ipxe-create`.
 
 !!! note
-    If there are multiple network interfaces on the installing machine, the user can use `ip=<interface>:dhcp` to specify the booting interface (e.g., `ip=eth1:dhcp`).
+    You can leverage dracut's `ip=` parameter to specify the booting interface and any network configurations dracut supports (e.g., `ip=eth1:dhcp`) if you have multiple network interfaces. See [`man dracut.cmdline`](https://man7.org/linux/man-pages/man7/dracut.cmdline.7.html) for more information.
```

Comment:
If you have multiple network interfaces, you can leverage dracut's `ip=` parameter to specify the booting interface and any other network configurations that dracut supports (e.g., `ip=eth1:dhcp`).